README.md

```diff
@@ -16,6 +16,4 @@ This is a disassembly and decompilation of Legend of Zelda: Ocarina of Time Mast
 It builds the following ROM:
 * zelda_ocarina_mq_dbg.z64 `md5: 717179476af84133b14ff73af87db57a`
 
```

Comment:
The checksum here should be updated to our new one as well right?
Also jenkins failed on this and I'm sure its because of the new checksum. This will probably be fixed after we merge this right?